app/controllers/learning_hours_controller.rb

```diff
@@ -22,6 +22,7 @@ def create
     respond_to do |format|
       if @learning_hour.save
         format.html { redirect_to volunteer_learning_hours_path(volunteer_id: current_user.id), notice: "New entry was successfully created." }
+        LearningHoursMailer.with(user: @user, casa_org: @casa_org, csv_data: csv_data).learning_hours_report_email.deliver_later
```

Comment:
Where is `@user` getting defined? Is this supposed to be `current_user` ?

Reply:
Yep that's supposed to be!